bazel/README.md

```diff
@@ -212,7 +219,7 @@ for how to update or override dependencies.
    in your shell for buildifier to work.
 1. `go get -u github.com/bazelbuild/buildtools/buildozer` to install buildozer. You may need to set `BUILDOZER_BIN` to `$GOPATH/bin/buildozer`
    in your shell for buildozer to work.
-1. `bazel build //source/exe:envoy-static` from the Envoy source directory.
+1. `bazel build -c opt --config=msvc-cl //source/exe:envoy-static` from the Envoy source directory.
```

Comment:
This step is technically cross-platform, so we shouldn't be adding --config=msvc-cl (nor -c really). I would recommend breaking this step up into non-Windows platforms (keep the original documentation) and Windows (add "-c opt").

msvc-cl should be the default, so I don't believe adding that param is necessary for getting a clean build on Windows

Reply:
Updated.